pr-data.csv

```diff
@@ -93,9 +93,9 @@ https://github.com/alibaba/fastjson,e05e9c5e4be580691cc55a59f3256595393203a1,.,c
 https://github.com/alibaba/fastjson,e05e9c5e4be580691cc55a59f3256595393203a1,.,com.alibaba.json.bvt.writeClassName.WriteClassNameTest_Map.test_list,ID,Opened,https://github.com/alibaba/fastjson/pull/3571,
 https://github.com/alibaba/fastjson,e05e9c5e4be580691cc55a59f3256595393203a1,.,com.alibaba.json.bvt.writeClassName.WriteDuplicateType.test_dupType,ID,Opened,https://github.com/alibaba/fastjson/pull/3533,
 https://github.com/alibaba/fastjson,e05e9c5e4be580691cc55a59f3256595393203a1,.,com.alibaba.json.bvt.writeClassName.WriteDuplicateType.test_dupType2,ID,Accepted,https://github.com/alibaba/fastjson/pull/2996,
-https://github.com/alibaba/innodb-java-reader,24a95d8005d480d460c3004fad89cfa295b9cbb4,innodb-java-reader,com.alibaba.innodb.java.reader.InnodbReaderBootstrapTest.testGenFillingRateHeatmap,ID,,,https://github.com/TestingResearchIllinois/idoft/issues/23
-https://github.com/alibaba/innodb-java-reader,24a95d8005d480d460c3004fad89cfa295b9cbb4,innodb-java-reader,com.alibaba.innodb.java.reader.InnodbReaderBootstrapTest.testGenLsnHeatmap,ID,,,https://github.com/TestingResearchIllinois/idoft/issues/23
-https://github.com/alibaba/innodb-java-reader,24a95d8005d480d460c3004fad89cfa295b9cbb4,innodb-java-reader,com.alibaba.innodb.java.reader.InnodbReaderBootstrapTest.testQueryAllDirectIO,ID,,,https://github.com/TestingResearchIllinois/idoft/issues/23
+https://github.com/alibaba/innodb-java-reader,24a95d8005d480d460c3004fad89cfa295b9cbb4,innodb-java-reader-cli,com.alibaba.innodb.java.reader.InnodbReaderBootstrapTest.testGenFillingRateHeatmap,ID,DeveloperFixed,,https://github.com/TestingResearchIllinois/idoft/issues/23
+https://github.com/alibaba/innodb-java-reader,24a95d8005d480d460c3004fad89cfa295b9cbb4,innodb-java-reader-cli,com.alibaba.innodb.java.reader.InnodbReaderBootstrapTest.testGenLsnHeatmap,ID,DeveloperFixed,,https://github.com/TestingResearchIllinois/idoft/issues/23
+https://github.com/alibaba/innodb-java-reader,24a95d8005d480d460c3004fad89cfa295b9cbb4,innodb-java-reader-cli,com.alibaba.innodb.java.reader.InnodbReaderBootstrapTest.testQueryAllDirectIO,ID,DeveloperFixed,,https://github.com/TestingResearchIllinois/idoft/issues/23
```

Comment:
The module name should be changed, but (1) the tests are OD not ID, and (2) checks all tests in com.alibaba.innodb.java.reader.InnodbReaderBootstrapTest (at least the seven already in pr-data.csv).